.github/workflows/test.yaml

```diff
@@ -66,3 +63,31 @@ jobs:
           make check
     - name: Test Go
       run: go test github.com/snapcore/snapd/...
+  spread:
+    needs: unit-tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        system:
+        - centos-8-64
+        - opensuse-15.1-64
+        - opensuse-tumbleweed-64
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install spread
+      run: |
+          cd /tmp
+          curl -s -O https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz
+          tar xzvf spread-amd64.tar.gz
+    - name: Run spread tests
+      env:
+          SPREAD_GOOGLE_KEY: ${{ secrets.SPREAD_GOOGLE_KEY }}
```

Comment:
Is this actually used?  It will result in an empty string for any PRs from people who are not members of snapcore.